### PR TITLE
Feat: Add to Wishlist and auth pages

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added 
- Wishlist page.
- Add to wishlist, remove from the wishlist, move the item from wishlist to cart.
- Auth pages added (Login, signup, forgot password)

chrome-capture (13)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,6 +18,10 @@ function App() {
         <Route path="/" element={<HomePage />} />
         <Route path="product" element={<Product />} />
         <Route path="cart" element={<Cart />} />
```
Comment (rehanmohiuddin) [inline src/App.js:20]: Add /routeName

File: src/Components/Cartlist/CartList.jsx [hunk 2]
```diff
@@ -43,7 +45,17 @@ const CartList = () => {
                 </div>
                 <div className="checkout-btn">
                   <button className="cart-btn">BUY</button>
-                  <button className="cart-btn">REMOVE</button>
+                  <button
+                    className="cart-btn"
+                    onClick={() =>
+                      dispatchCart({
+                        type: "REMOVE_FROM_CART",
```
Comment (rehanmohiuddin) [inline src/Components/Cartlist/CartList.jsx:52]: Add constants variable for removefromcart just a suggestion

File: src/index.js [hunk 3]
```diff
@@ -8,23 +8,26 @@ import { BrowserRouter } from "react-router-dom";
...
-        <ProductProvider>
-          <CategoryProvider>
-            <BrowserRouter>
-              <App />
-            </BrowserRouter>
-          </CategoryProvider>
-        </ProductProvider>
-      </FilterProvider>
-    </CartProvider>
+    <WishProvider>
```
Comment (rehanmohiuddin) [inline src/index.js:18]: Just a suggestion make one index file for all context providers and wrap all in one then import in index.css

**Review by rehanmohiuddin — CHANGES_REQUESTED** [review]
Suggested some changes

File: src/index.js (hunk 3 above)
Comment (Prabhav-Sharma) [reply src/index.js:18]: +1

**Review by Prabhav-Sharma — APPROVED** [review]
Code is clean and easy to navigate!

**Review by Veekshith-Thoram — APPROVED** [review]

File: src/Components/Cartlist/CartList.jsx (hunk 2 above)
Comment (abhijeetekad) [reply src/Components/Cartlist/CartList.jsx:52]: okk

File: src/index.js (hunk 3 above)
Comment (abhijeetekad) [reply src/index.js:18]: Ok... Will do that while refactoring code for sure. thanks for the suggestion.

File: src/App.js (hunk 1 above)
Comment (abhijeetekad) [reply src/App.js:20]: ok...

**Pull request merged**